source/common/http/http2/codec_impl.h

```diff
@@ -335,6 +335,9 @@ class ConnectionImpl : public virtual Connection, protected Logger::Loggable<Log
   using ServerStreamImplPtr = std::unique_ptr<ServerStreamImpl>;
 
   ConnectionImpl* base() { return this; }
+  // NOTE: Always use runtime nullptr checks against the return value of this function. There are
```

Comment:
nit: mind switching this to something like
Always check for getStream returning nullptr because ...
I associate runtime with the Envoy runtime class :-P

Reply:
Sure, I'll adjust the wording.

Reply:
I've now changed the comment.